dockers/docker-base/Dockerfile.j2

```diff
@@ -25,12 +25,14 @@ RUN apt-get clean && apt-get update
 RUN apt-get -y install \
     rsyslog            \
     supervisor         \
-    vim-tiny           \
+    vim                \
```

Comment:
Keep vim.tiny for docker-base. But put .vimrc and make link from vim.tiny to vim.